test/unit/modal.ts

```diff
@@ -0,0 +1,102 @@
+import { utils } from '@storefront/core';
+import Modal from '../../src/modal';
+import suite from './_suite';
+
+suite('Modal', ({ expect, spy, stub }) => {
+  let modal: Modal;
+
+  beforeEach(() => modal = new Modal());
+
+  describe('constructor()', () => {
+    describe('props', () => {
+      describe('autoOpen', () => {
+        it('should set default value', () => {
```

Comment:
if we only have the one `props` value I think we can drop the `describe('autoOpen')` in favour of `describe('props')`